app/main.go

```diff
@@ -156,10 +160,22 @@ func newConfig(ctx context.Context) (*tsbridge.Config, error) {
 		return nil, fmt.Errorf("Could not parse COUNTER_RESET_INTERVAL: %v", err)
 	}
 
+	storageEngine := os.Getenv("STORAGE_ENGINE")
+	switch storageEngine {
+	case "datastore":
+		storageManager = datastore.New()
+	case "":
```

Comment:
As nowhere in the default configs we explicitly assign STORAGE_ENGINE,  os.Getenv("STORAGE_ENGINE") will return an empty string and this will lead to "defaulting to GAE datastore".  

This is fine, but let's make sure we explicitly assign STORAGE_ENGINE=datastore in app.yaml and maybe also reference to it in README.md #global-settings , so that potential customers of ts-bridge are aware, that there is a setting like that, even though it's currently allowing just 1 option to choose from.

Reply:
Done.